### PR TITLE
[CM-574] Initial Setup

### PR DESCRIPTION
**Pull request by mschuwalow**

### CODE REVIEW

File: .circleci/.circleci/.circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,247 @@
+version: 2.1
```
Comment (jtatarik) [inline .circleci/.circleci/.circleci/config.yml:1]: I think this belongs to `.circleci/.circleci/.circleci/.circleci/.circleci/.circleci`

Comment (mschuwalow) [reply .circleci/.circleci/.circleci/config.yml:1]: oh, fingers were fat during copy pasta :D

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# live-connect-handlers
-Default Implementation of handlers for live-connect
+# live-connect-common
```
Comment (3link) [inline README.md:1]: Should it be handlers after all?

Comment (mschuwalow) [reply README.md:1]: good catch, fixed

**Pull request merged**